src/mbgl/renderer/render_source.hpp

```diff
@@ -70,6 +70,14 @@ class RenderSource : protected TileObserver {
     virtual std::vector<Feature>
     querySourceFeatures(const SourceQueryOptions&) const = 0;
 
+    virtual mapbox::util::variant<NullValue, Value, std::vector<Feature>>
```

Comment:
`Value` is a variant that includes `NullValue`. this can be reduced to `variant<Value, FeatureCollection>`. It should also be aliased in the appropriate util header, maybe `geojson.hpp`?

Reply:
I removed Null type from variant, however, I'm not sure if alias for `mapbox::util::variant<Value, FeatureCollection>` is common and can be moved to `geojson.hpp`

Reply:
The `Feature` type returned by `queryRenderedFeatures` is also present in the `geojson.hpp` header. It should be fine to add this alias there as well.

Reply:
Moved alias to `include/mbgl/util/geojson.hpp`.